### ISSUE TITLE
URLs generated by "test run" command are not escaped

### ISSUE THREAD

**Issue by jmoody** [Microsoft/appcenter-cli#442]
### Reproduce

Requires:

* macOS
* Xcode
* local clone of https://github.com/Microsoft/AppCenter-Test-XCUITest-Extensions/
* add Flowers iOS to your appcenter account
* generate a device set

```
$ bin/make/build-for-testing.sh arm
$ SERIES="Spaces In Name" bin/make/appcenter.sh < device set >
```

### Expected

`appcenter appcenter test run` to print a valid URL to stdout at the end of the job.

### Found

The URL is invalid because it contains spaces.

Example:

```
https://appcenter.ms/orgs/org/apps/Flowers/test/series/DYLD Bootstrap.dylib/runs/udid
```

### Notes

This is a problem for CI systems that attempt to inject this URL.

**Comment by amchew** [comment]
Hi @jmoody, an update, we're merged the fix in the [PR](https://github.com/Microsoft/appcenter-cli/pull/443) and will be releasing the fix soon.

**Comment by amchew** [comment]
Hi @jmoody this issue should have been fixed with the PR #443, and released in the new CLI v1.1.0 version today. Could you install the new CLI version, and check if this issue is resolved? Thanks.

**Comment by amchew** [comment]
Hi @jmoody this issue should have been fixed with the PR #443, please test it out. Feel free to reach out again if you have any questions! I'll be closing this ticket for now.

Amanda
